kolena/metrics/_geometry.py

```diff
@@ -345,12 +339,12 @@ def match_inferences_multiclass(
 
     confused = []
     for gt, inf in confused_matches.matched:
-        if gt.label != inf.label:
+        if gt.label != inf.label:  # type: ignore[union-attr]
```

Comment:
Is there a way to accommodate optional labels without muting mypy?

Reply:
Guarding with an `assert` works if we prefer that approach